content/chat/setup.textile

```diff
@@ -59,6 +60,11 @@ import * as Ably from 'ably';
 import ChatClient from '@ably/chat';
 ```
 
+```[react]
+import { ChatClientProvider } from '@ably/chat/react';
```

Comment:
They'll still need the `ChatClient` import from `@ably/chat` to be able to instantiate React